frontend/src/components/OdhExploreCardTypeBadge.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Tooltip } from '@patternfly/react-core';
+import { OdhApplicationCategory } from '../types';
+
+type OdhExploreCardTypeBadgeProps = {
+  category: OdhApplicationCategory;
+};
+
+const OdhExploreCardTypeBadge: React.FC<OdhExploreCardTypeBadgeProps> = ({ category }) => {
+  let content;
+
+  if (category === OdhApplicationCategory.RedHatManaged) {
+    content = 'Red Hat managed software is hosted on Red Hat’s OSD cluster';
+  } else if (category === OdhApplicationCategory.PartnerManaged) {
+    content = 'Partner managed software is hosted on the ISV’s cloud service';
+  } else if (category === OdhApplicationCategory.SelfManaged) {
+    content =
+      'Self-managed software is installed to a RHODS cluster, but does not support upgrade testing, alerting, or other features of externally managed software';
+  } else {
```

Comment:
Small nit -- I get what you're doing here works... but I'd recommend actually doing this:

```
if - case A; set tooltip to 'A'
else if - case B; set tooltip to 'B'
else if - case C; set tooltip to 'C'

if !tooltip; return
```
That way if we add another case that doesn't set the tooltip for whatever reason we don't try to render a Tooltip with content.

It's the same thing right now, but it separates the category statements from the return -- so we can manage the fall-through case a bit less "in with the rest"